tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -88,31 +88,11 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Variables", "[DataStructures][Unit]") {
   CHECK(v2 == v3);
 
   const std::string expected_output =
-      "--Symmetry:  (1)\n"
-      "--Types:     (Spatial)\n"
-      "--Dims:      (3)\n"
-      "--Locations: (Up)\n"
-      "--Frames:    (Grid)\n"
-      " T(0)=(-4)\n"
-      "     Multiplicity: 1 Index: 0\n"
-      " T(1)=(-4)\n"
-      "     Multiplicity: 1 Index: 1\n"
-      " T(2)=(-4)\n"
-      "     Multiplicity: 1 Index: 2\n"
-      "--Symmetry:  ()\n"
-      "--Types:     ()\n"
-      "--Dims:      ()\n"
-      "--Locations: ()\n"
-      "--Frames:    ()\n"
-      " T()=(-3)\n"
-      "     Multiplicity: 1 Index: 0\n"
-      "--Symmetry:  ()\n"
-      "--Types:     ()\n"
-      "--Dims:      ()\n"
-      "--Locations: ()\n"
-      "--Frames:    ()\n"
-      " T()=(-3)\n"
-      "     Multiplicity: 1 Index: 0";
+      "T(0)=(-4)\n"
+      "T(1)=(-4)\n"
+      "T(2)=(-4)\n"
+      "T()=(-3)\n"
+      "T()=(-3)";
```

Comment:
This is difficult to understand with all the tensors run together.  Perhaps print out the label for each one before the values?

Reply:
I agree. The first thing I've decided to change is there is an extra space between each tensor. Secondly, if the tag has a label it is printed, and nothing is printed if it doesn't. What are people's thoughts on this?

Reply:
I think we should either require tags to have labels or ignore the labels and just use `pretty_type::short_name`.

Reply:
I'm in favor of just using `pretty_type::short_name`. This should be more than enough for diagnostics. If template parameters are important there's always TypeDisplayer.